linode/manager
# chore: Move new OAuth e2e tests into 'core' directory

## Description 📝
This moves our OAuth E2E tests into the "core" test suite directory -- they were accidentally left out because of the timing of that PR and the PR which added the "core" directory, so just circling back to take care of that.

## How to test 🧪
1. Confirm that the OAuth E2E tests are included in the automated run for this PR, and that they pass
